bears/haskell/HaskellLintBear.py

```diff
@@ -19,7 +21,8 @@ class HaskellLintBear:
     """
 
     LANGUAGES = {'Haskell'}
-    REQUIREMENTS = {DistributionRequirement(apt_get='hlint')}
+    REQUIREMENTS = {CabalRequirement(package='hlint', version='1.9.35'),
```

Comment:
You can now use [AnyOneOfRequirements](https://gitlab.com/coala/package_manager/blob/master/dependency_management/requirements/AnyOneOfRequirements.py) to wrap these two requirements in an OR statement.